lib/baza_models/helpers/ransacker_helper.rb

```diff
@@ -53,7 +53,7 @@ def bm_sort_link(ransacker, attribute, label = nil)
       new_params[:q][:s] = sort_asc
     end
 
-    href = url_for(new_params)
+    href = url_for(new_params.permit!)
```

Comment:
Shouldn't this be done on line 45 so it is done the same way as on line 5?

Reply:
Does not work. Raises `ActionController::UnfilteredParameters in AwesomeTranslations::HandlerTranslations#index`

Only works when params already includes `q`& `s`. 

```
    new_params = params.clone
    new_params[:q] ||= {}
    new_params[:q][:s] ||= nil
    new_params.permit!
```

Would work too. As new_params will have `q`& `s` when called permit!

Reply:
Isn't that because it should be moved to directly after the `.clone`?

```ruby
    new_params = params.clone
    new_params.permit!
    new_params[:q] ||= {}
    new_params[:q][:s] ||= nil
```

Reply:
nope. Still does not work..

`permit!` has to be called after 
```ruby
  new_params[:q] ||= {}
  new_params[:q][:s] ||= nil
```

When the structure changes it has to be permitted again